README.md

```diff
@@ -57,7 +34,7 @@ To build for esp or host, do the following commands. Supplying the to cmake will
 ````
 mkdir build
 cd build
-cmake .. -DCMAKE_BUILD_TYPE=Debug <-DCMAKE_TOOLCHAIN_FILE=../cmake/esp-idf/toolchain-esp32.cmake -DTARGET=esp32>
+cmake .. -DCMAKE_BUILD_TYPE=Debug <-DCMAKE_TOOLCHAIN_FILE=../cmake/esp-idf/toolchain-esp32.cmake>
```

Comment:
When esp variables are sourced, the system python is changed for the one provided by esp. The problem is that catkin is not installed in that python so when we build for catkin we have to provide the default system python. Maybe this should be added to the readme with the argument to do so. `-DPython3_EXECUTABLE=/usr/bin/python3`

Reply:
Done